optaplanner-operator/pom.xml

```diff
@@ -27,7 +28,12 @@
     <version.io.quarkus.override>2.15.0.Final</version.io.quarkus.override>
   </properties>
 
-  <!-- TODO: Remove when OptaPlanner no longer supports Quarkus 2.13 -->
+  <modules>
+    <module>optaplanner-operator-impl</module>
```

Comment:
The previous comment opens the possibility to keep the original name here: `optaplanner-operator`.

Reply:
a60c0a97